shared/src/main/java/org/eclipse/steady/shared/util/VulasConfiguration.java

```diff
@@ -298,14 +299,20 @@ private void addConfiguration(Configuration _cfg, String _source) {
    */
   public void sanitize(Configuration _cfg) {
     final Iterator<String> i = _cfg.getKeys();
+    List<String> keys_to_remove = new ArrayList<String>();
     while (i.hasNext()) {
       final String k = i.next();
       final Matcher m = KEY_PATTERN.matcher(k);
       if (!m.matches()) {
         getLog().warn("Configuration key [" + k + "] removed due to illegal characters");
-        _cfg.clearProperty(k);
+        keys_to_remove.add(k);
+        //_cfg.clearProperty(k);
```

Comment:
Do you think it is necessary to keep those two lines... ;)

Reply:
i agree, just an old habit :)